LottieSample/src/main/kotlin/com/airbnb/lottie/samples/PlayerFragment.kt

```diff
@@ -512,13 +522,12 @@ class PlayerFragment : Fragment() {
     private fun beginDelayedTransition() = TransitionManager.beginDelayedTransition(container, transition)
 
     companion object {
+
         const val EXTRA_ANIMATION_ARGS = "animation_args"
 
-        fun forAsset(args: CompositionArgs): Fragment {
-            return PlayerFragment().apply {
-                arguments = Bundle().apply {
-                    putParcelable(EXTRA_ANIMATION_ARGS, args)
-                }
+        fun newInstance(args: CompositionArgs) = PlayerFragment().apply {
```

Comment:
change function name to make it follow factory design pattern convention.
use single expression function make the code more concise.
reference: [http://kotlinlang.org/docs/reference/functions.html#single-expression-functions](http://kotlinlang.org/docs/reference/functions.html#single-expression-functions)